zipkin-cassandra/src/main/scala/com/twitter/cassie/tests/util/FakeCassandra.scala

```diff
@@ -254,7 +255,15 @@ class FakeCassandra extends Cassandra.Iface {
     }
   }
 
-  def login(auth_request: AuthenticationRequest) { throw new UnsupportedOperationException }
+  def login(auth_request: AuthenticationRequest) {
+    if(auth_request.credentials("username") == Auth.username &&
+       auth_request.credentials("username") == Auth.username){
```

Comment:
why have an object of `Auth` cluttering the `util.*` namespace? why not use a `val` in this `FakeCassandra` object instead?